app/pages/sections/additional-services/select/recipient/controller.js

```diff
@@ -0,0 +1,14 @@
+import { getData } from 'buying-catalogue-library';
+import { getContext } from './contextCreator';
+import { getEndpoint } from '../../../../../endpoints';
+import { logger } from '../../../../../logger';
+
+export const getAdditionalServiceRecipientPageContext = params => getContext(params);
+
+export const getSolution = async ({ solutionId, accessToken }) => {
```

Comment:
We should have an `api` directory within helpers. There is a folder for `bapi` can you add this function there please. and the test you have in the controller there too. This should start tidying up the controllers a bit and also have a central place where we make api calls across all the pages.

Reply:
on second thoughts I don't think we want to be calling getSolution at all. Think you want to be calling getCatalogueItem. That should already exist. So we can remove this and just call that function instead